src/Engine/ProtoCore/Reflection/Mirror.cs

```diff
@@ -933,6 +933,22 @@ public List<string> GetArgumentNames()
                 return argNames;
             }
 
+            // List<ArgumentInfo> GetArguments() is used to get "Arguments"--> so that can use "Arguments" to get ArgumentsType;
+            //original method is  public List<ProtoCore.Type> GetArgumentTypes()
+            public List<ArgumentInfo> GetArguments()
```

Comment:
I cannot figure out a method to use ".Arguments" ; 
it is called in a format of " method[0].GetArguments(); "  . if I changed it to method[0].Arguments  1) method[0] .Arguments does not exsit ;2) method[0] can get method[0].ArgumentsList;

ps: originally there exist a method: public List<protocore.type> GetArgumentTypes(){...}
It is like someone can also choose to use ProcNode.typeList instead of using this method.